docs/sources/userguide/plugins.md

```diff
@@ -0,0 +1,157 @@
+Docker Plugins MVP
+==================
+
+Plugins provide a mechanism for hooking into the Docker engine to extend its behavior.
+
+HTTP over a local UNIX socket is used to communicate between Docker and its plugins.
+
+Plugins are distributed as containers.
+
+This is targeted at Docker 1.7.
+
+# Usage
+
+Plugins can be loaded using a special `docker plugin load` command, as follows:
```

Comment:
I guess this is missing context. There is a very common pattern in deployments of "system" containers or "agents", whether custom or from a third party that do various things for the system, but is not part of the main compute payload of a host. 

Many examples are candidates for "plugins" but not all of them necessarily need to hook into Docker extension points. They just need to 1) always run on Docker boot, 2) likely have a default restart policy, 3) not show in main `docker ps`. Their management semantics match plugins but are not always plugins. For example, Docker Swarm is an agent. 

Another example of this pattern shows up in RancherOS, how they've separated system containers from user containers. Our terminology is "agent".